pkg/endpoints/credentials_test.go

```diff
@@ -63,7 +65,7 @@ func TestCredentials(t *testing.T) {
 	// CREATE credential accesstoken
 	t.Log("CREATE credential accesstoken")
 	createCredentialTest(namespace, accessTokenCred, "", r, t)
-	if !r.checkSecretInSa(namespace, "default", "credentialaccesstoken", t) {
+	if !r.checkSecretInSa(namespace, "sa1", "credentialaccesstoken", t) {
```

Comment:
This looks like it's checking that accessTokenCred is to be patched onto sa1. Is that right? Should there also be a test that credentialuserpass is patched onto the default service account?

Reply:
Yes, it is tested around line 75.